exporter/oci/export.go

```diff
@@ -153,7 +153,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source,
 	resp["containerimage.digest"] = desc.Digest.String()
 	if v, ok := desc.Annotations["config.digest"]; ok {
 		resp["containerimage.config.digest"] = v
-		delete(desc.Annotations, "config.digeest")
+		delete(desc.Annotations, "config.digest")
```

Comment:
IMO keys like this should be constants rather than literal strings to prevent exactly this kind of bug.

Reply:
Good point, I've created the constants in the `exptypes` package as it seems the most appropriate one.